content/community/resources/market-reports/industrial-iot-world-2019.md

```diff
@@ -0,0 +1,9 @@
++++
+date = "2019-11-12"
+title = "Industrial IoT World: Seeking ROI, Disruptive Technologies like AR Take Center Stage"
+seo_title = "Industrial IoT World: Seeking ROI, Disruptive Technologies like AR Take Center Stage"
+link = "/community/resources/market-reports/pdf/451_Reprint_IndustrialIoTWorld_12NOV2019.pdf"
```

Comment:
Let's create friendlier urls. 

For example:
/community/resources/market-reports/pdf/industrial-iot-world-2019.pdf
/community/resources/market-reports/images/industrial-iot-world-2019.png

Reply:
Done